Libraries/LibLine/Editor.cpp

```diff
@@ -635,6 +634,11 @@ void Editor::handle_read_event()
     Vector<unsigned, 4> csi_parameters;
     Vector<u8> csi_intermediate_bytes;
     u8 csi_final;
+    enum CSIMod {
+        Shift = 1,
+        Alt = 2,
+        Ctrl = 4,
+    };
```

Comment:
maybe make this an `enum class`? you seem to be using it qualified anyway.

Reply:
Then I need to explicitly cast it to int.